docs-en/chapter_computational_complexity/summary.md

```diff
@@ -2,48 +2,48 @@
-- We usually focus only on the worst-case space complexity, i.e., the space complexity of the statistical algorithm with the worst-case input data and the worst-case running time complexity.
-- Common space complexities are $O(1)$, $O(\log n)$, $O(n)$, $O(n^2)$ and $O(2^n)$ in descending order.
+- Space complexity serves a similar purpose to time complexity and is used to measure the trend of space occupied by an algorithm as the data volume increases.
+- The memory space associated with the operation of an algorithm can be categorized into input space, temporary storage space, and output space. Normally, the input space is not considered when determining space complexity. The transient space can be classified into instruction space, data space, and stack frame space, and the stack frame space usually only affects the space complexity for recursion functions.
+- We mainly focus on the worst-case space complexity, i.e., computing the space complexity with the worst input data and runtime.
+- Common space complexities are $O(1)$, $O(\log n)$, $O(n)$, $O(n^2)$ and $O(2^n)$ from lowest to highest.
 
 ### Q & A
 
 !!! question "Is the space complexity of tail recursion $O(1)$?"
 
-    Theoretically, the space complexity of a tail recursion function can be optimized to $O(1)$ . However, most programming languages (e.g. Java, Python, C++, Go, C#, etc.) do not support automatic optimization of tail recursion, so the space complexity is usually considered to be $O(n)$ .
+    Theoretically, the space complexity of a tail recursion function can be optimized to $O(1)$. However, most programming languages (e.g., Java, Python, C++, Go, C#, etc.) do not support auto-optimization for tail recursion, so the space complexity is usually considered as $O(n)$.
 
 !!! question "What is the difference between the terms function and method?"
 
-    A function can be executed independently and all arguments are passed explicitly. A method is associated with an object, is implicitly passed to the object that calls it, and can manipulate data contained in instances of the class.
+    A *function* can be executed independently, and all arguments are passed explicitly. A *method* is associated with an object and is implicitly passed to the object that calls it, and can manipulate data contained in instances of the class.
 
-    Here are a few common programming languages to illustrate.
+    Let's illustrate with a few common programming languages.
 
-    - C is a procedural programming language and has no object-oriented concepts, so it has only functions. However, we can simulate object-oriented programming by creating structures, and the functions associated with structures are equivalent to methods in other languages.
```

Comment:
C can be strong **C**, like the other languages.

Reply:
@thomasq0 I don't think it is necessary to bold the language name C because it doesn't cause ambiguity.